docs/config.md

```diff
@@ -452,6 +452,12 @@ If you would like to use Scalar, the integration manager maintained by Element,
 }
 ```
 
+For widgets in general (from an integration manager or not) there is also:
+
+-   `default_widget_container_height`
+
+This controls the height that the top widget panel initially appears as and is the height in pixels, default 280.
```

Comment:
This doesn't match the value in the config sample, is the default `280` or `280px`?

Reply:
Oops, good spot, thanks